### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by thisisamanmishra** [community/community#60857] [GitHub Education]
### Hi everyone,

Actually, my student developer pack was over and now, I've to reapply for it but now, this time they ask for College Email id given to me having College Domain but my college doesn't provide email id and because of this everytime I apply my application gets rejected. So, what should I do now??

**Comment by notscar** [comment]
You can't do nothing, i've reapplied for 8 times now, after it expiring providing screenshots of the school portal, providing gmail screenshots since we can't send or receive emails from outside or to outside the organization, i've shown a document that shows my details and academic year, changed my github name to my full name, and it's still rejected, since there was no problem with the application they then sent me to https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/why-wasnt-my-application-to-global-campus-for-students-approve, that makes no sense, they are essentially gate keeping, enjoy github service i guess.

**Comment by nonvenomous** [comment]
Same issue here, [Required mail from institution but institution does not provide it #60650](https://github.com/orgs/community/discussions/60650).
No known solution so far.

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.

For everyone else If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! 👍
